lisa/tools/mkswap.py

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from lisa.executable import Tool
+
+
+class MkSwap(Tool):
+    @property
+    def command(self) -> str:
+        return "mkswap"
```

Comment:
The new tool should be added into `__init__.py`, so it's easy to reference.

Reply:
oops. missed it